pages/lists/[listId]/activity.tsx

```diff
@@ -31,16 +31,23 @@ interface ContributorListPageProps {
   isOwner: boolean;
 }
 
+export type ServerFilterParams = {
+  listId: string;
+  range?: string;
+  limit?: string;
+};
+
 export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
   const supabase = createPagesServerClient(ctx);
 
   const {
     data: { session },
   } = await supabase.auth.getSession();
   const bearerToken = session ? session.access_token : "";
-  const { listId, range: rawRange } = ctx.params as { listId: string; range: string };
+  const { listId, range: rawRange, limit: RawLimit } = ctx.params as ServerFilterParams;
```

Comment:
Refactor (blocking): `RawLimit` should be camel case as it's a JavaScript variable and not a type.

```suggestion
  const { listId, range: rawRange, limit: rawLimit } = ctx.params as ServerFilterParams;
```